src/components/settings/settings.tsx

```diff
@@ -65,12 +69,149 @@ class Settings extends React.Component<Props, State> {
             </Box>
           </Toolbar>
         </AppBar>
-        <Box p={ 3 }>
-          {/* Settings content here */}
-        </Box>
+        { this.accordionMenu() }
+        {/* Settings content here */}
       </>
     );
   }
+
+  /**
+   * Method for settings accordion menu
+   */
+  private accordionMenu() {
+
+    return (
+      <Box p={ 3 }>
+        { this.accordionMenuTransportation() }
+        { this.accordionMenuMedical() }
+        { this.accordionMenuCustom() }
+      </Box>
+    );
+  }
+
+  /**
+   * Method for transportation accordion menu item
+   */
+  private accordionMenuTransportation = () => {
+    return (
+      <Accordion defaultExpanded={ true }>
+        <AccordionSummary expandIcon={ <ExpandMoreIcon /> }>
+          <Typography>{ strings.settingsDrawer.transportation }</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Typography>{ strings.settingsDrawer.descriptionTransportation }</Typography>
+          <FormGroup>
+            { this.menuCheckbox(strings.walking) }
+            { this.menuCheckbox(strings.wheelerchair) }
+            { this.menuCheckbox(strings.walking) }
+            { this.menuCheckbox(strings.wheelerchair) }
+          </FormGroup>
+        </AccordionDetails>
+      </Accordion>
+    );
+  }
+
+  /**
+   * Method for medical accordion menu item
+   */
+  private accordionMenuMedical = () => {
+    return (
+      <Accordion defaultExpanded={ true }>
+        <AccordionSummary expandIcon={ <ExpandMoreIcon /> }>
+          <Typography>{ strings.settingsDrawer.medical }</Typography>
+        </AccordionSummary>
+        <AccordionDetails>
+          <Typography>{ strings.settingsDrawer.descriptionMedical }</Typography>
+          <FormGroup>
+            { this.menuCheckbox(strings.medicalConditions.asthma) }
+            { this.menuCheckbox(strings.medicalConditions.ischaemicHeartDisease) }
+            { this.menuCheckbox(strings.medicalConditions.copd) }
+          </FormGroup>
+        </AccordionDetails>
+      </Accordion>
+    );
+  }
+
+  /**
+   * Method for rendering checkbox
+   * @param label checkbox label string
```

Comment:
```suggestion
   * Method for rendering checkbox
   *
   * @param label checkbox label string
```

It is good to separate description and parameters here